MapboxNavigation/DelayInformationView.swift

```diff
@@ -0,0 +1,34 @@
+import UIKit
+
+class DelayInformationView: UIView {
```

Comment:
I don't really like creating `DelayInformationView` which is then later converted to `UIImage` so that it's possible to use it in `MGLSymbolStyleLayer` (there are some additional issues like text overlapping etc). 

On the other hand seems that `MGLPointAnnotation` will not be performant enough when using large amount of delay annotations. It'll also require implementation of `MGMapViewDelegate` methods inside `NavigationMapView`.

@1ec5 if you have any better ideas please let me know.

Reply:
@MaximAlien have you seen https://docs.mapbox.com/mapbox-gl-js/style-spec/layers/#layout-symbol-icon-text-fit? Might be useful here.

Reply:
/cc @langsmith ^^^ in case we've not considered this for the Android impl.